app-misc/ddcutil/ddcutil-1.2.1.ebuild

```diff
@@ -0,0 +1,109 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
Could we bump this to `EAPI=8` please while we are touching the package anyway.

Reply:
Updated.